senderworker/riff-raff.yaml

```diff
@@ -0,0 +1,23 @@
+stacks: [mobile-notifications-workers]
+regions: [eu-west-1]
+allowedStages:
+  - CODE
+  - PROD
+deployments:
+  senderworker-cfn:
+    type: cloud-formation
+    app: sender-worker
```

Comment:
Would we expect to see logs under this `appName` in kibana?

Reply:
Not yet.  The logging configuration will be wired up in [LIVE-4736](https://theguardian.atlassian.net/browse/LIVE-4736).  After that, its logs are under this `appName`